pkg/placement/workload-projector.go

```diff
@@ -429,8 +374,15 @@ func (wpd *wpPerDestination) getDynamicDuoLocked(gr metav1.GroupResource, apiVer
 					opts.LabelSelector = opts.LabelSelector + "," + justMineStr
 				}
 			})
-		wpd.namespaceClient = wpd.wp.nsClusterClient.Cluster(mbwsCluster.Path())
-		wpd.namespacePreInformer = wpd.wp.nsClusterPreInformer.Cluster(mbwsCluster)
+		mbsClient, err := k8sclient.NewForConfig(config)
```

Comment:
This is not the only client of the MBS. I would name this variable something more distinctive, like `kubeClient`.